plugin/src/main/kotlin/com/moengage/internal/repository/NexusRepository.kt

```diff
@@ -10,6 +10,7 @@ import com.moengage.internal.model.NexusStagingRepositoryData
 import com.moengage.internal.repository.network.NexusService
```

Comment:
`currentRetryCount` could be named `retryCount` across the file.

Reply:
done